SHAPETEST/StartShapeTestLosProbe.md

```diff
@@ -23,6 +23,7 @@ enum TraceFlags
   IntersectWater = 32,
   Unknown = 128,
   IntersectFoliage = 256,
+  Unkown = 511, (used most)
   IntersectEverything = 4294967295
```

Comment:
I believe this should be `-1`, but it incorrectly got displayed as `4294967295` (`0xFFFFFFFF`). I have seen that happen multiple times in other unrelated cases. Maybe a test with the value being `-1` rather than  (`4294967295`) should be done in order to verify.

Reply:
I'm working on raycasting so I'll check it out

Reply:
Can confirm now, -1 intersects everything